### ISSUE TITLE
你好，训练时 ，遇到第一层Conv_weight的值是NAN

### ISSUE THREAD

**Issue by LouiValley** [hzy46/fast-neural-style-tensorflow#39]
在训练的时候，我print了第一层权重的值，发现训练到后来出现NAN。
我把learning rate 调小了也没有什么用处。
batch size =16
learning rate=1e-5
